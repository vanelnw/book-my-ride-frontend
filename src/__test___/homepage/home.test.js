import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';

import Home from '../../components/home/Home';
import store from '../../redux/store';

it('Cars renders correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Router>
        <Home />
      </Router>
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
