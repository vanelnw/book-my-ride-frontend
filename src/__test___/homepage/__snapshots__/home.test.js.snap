// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cars renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="headline-cont"
  >
    <h1>
      Latest Models
    </h1>
    <span>
      Please select a car
    </span>
  </div>
  <div
    className="cars-container"
  />
</div>
`;
